src/bridge/ui_commands.rs

```diff
@@ -112,25 +112,25 @@ impl UiCommand {
                 if unregister_rightclick() {
                     let msg = "Could not unregister previous menu item. Possibly already registered or not running as Admin?";
                     nvim.err_writeln(msg).await.ok();
-                    error!("{}", msg);
```

Comment:
I'm curious about this change. Wouldn't we want to log these as errors to the console if the registration fails?